examples/subscriptions/src/index.ts

```diff
@@ -55,7 +64,7 @@ const resolvers = {
           await wait(1000)
         }
       },
-      resolve: (payload) => payload,
+      resolve: (payload: any) => payload,
```

Comment:
not so sure if this is the right thing to do cause rest worked well with codegen